prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -541,6 +552,11 @@ void rxr_pkt_handle_send_completion(struct rxr_ep *ep, struct fi_cq_data_entry *
 	case RXR_DC_EAGER_TAGRTM_PKT:
 		/* completion will be written upon receving the receipt packet, thus no action to be taken her */
 		break;
+	case RXR_DC_MEDIUM_MSGRTM_PKT:
+	case RXR_DC_MEDIUM_TAGRTM_PKT:
+		rxr_pkt_handle_dc_long_rtm_send_completion(ep, pkt_entry);
```

Comment:
should be ```rxr_pkt_handle_dc_medium_rtm_send_completion```

Reply:
`RXR_DC_MEDIUM_TAGRTM_PKT` and `RXR_DC_LONG_PKT` will share the same send completion functions. We also do this for non dc protocols.

Reply:
Even though the function implementation is the same, it is more readable that they have separate names. The non-dc situation is a mistake, I am going to open a PR to fix it.

Reply:
OK. Thank you.